libs/blocks/bulk-publish-v2/components/job-info.js

```diff
@@ -128,18 +143,30 @@ class JobInfo extends LitElement {
       <div class="job-info">
         <div class="process">
           <span class="topic">${topic}</span>
+          ${this.copied ? html`
+            <div class="copied">
+              Copied ${this.copied} to clipboard
+            </div>
+            ` : nothing}
           ${invocationId ? html`
             <img
               title="Copy Job Invocation Id"
               src="${copyIcon}"
               class="job-id-link"
               @click=${this.copyInvocationId} />
           ` : nothing}
-          ${this.copiedInvocationId ? html`
-            <div class="job-id-copied">
-              Copied to clipboard!
-            </div>
+          ${invocationId ? html`
+            <img
+              title="Share this Job"
```

Comment:
Should this be `Copy job link` instead. Since both icons are copying to clipboard the alt text reads a little funny when one says copy, and the other says share. Just a thought when i tested the UI.

Reply:
Cool yeah will update that